packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MarkForSale/index.tsx

```diff
@@ -300,10 +300,10 @@ const MarkForSale: React.FC<Props> = (props) => {
                       style={
                         saleType === 'timed-auction'
                           ? {
-                              background: colors.blue000,
-                              border: `1px solid ${colors.blue600}`
+                              background: PaletteColors['blue-000'],
+                              border: `1px solid ${PaletteColors['blue-600']}`
                             }
-                          : { border: `1px solid ${colors.grey100}` }
+                          : { border: `1px solid ${PaletteColors['grey-100']}` }
```

Comment:
Won't really argue with all of this, but seems a nice place to use the `css` import from `styled-components`